Components/Optimizers/AdaptiveStochasticLBFGS/elxAdaptiveStochasticLBFGS.hxx

```diff
@@ -240,7 +240,7 @@ void AdaptiveStochasticLBFGS<TElastix>
   double gradientMagnitudeTolerance = 0.000001;
```

Comment:
Interesting... shouldn't the comment be closed here, at the end of line 239, already? @mstaring ?

Reply:
That's whay I thought as well, but apparently it's unused?

Reply:
@haampie Marius and I just discussed: the comment line should be closed at the end of line 239. The code that does `m_Configuration->ReadParameter` should be enabled. Can you please fix that as well?